pkg/services/ngalert/api/api_provisioning.go

```diff
@@ -124,9 +124,9 @@ func (srv *ProvisioningSrv) RoutePutPolicyTree(c *contextmodel.ReqContext, tree
 }
 
 func (srv *ProvisioningSrv) RouteResetPolicyTree(c *contextmodel.ReqContext) response.Response {
-	tree, err := srv.policies.ResetPolicyTree(c.Req.Context(), c.SignedInUser.GetOrgID())
+	tree, err := srv.policies.ResetPolicyTree(c.Req.Context(), c.SignedInUser.GetOrgID(), alerting_models.ProvenanceAPI)
```

Comment:
Let's use `provenance := determineProvenance(c)` instead of `alerting_models.ProvenanceAPI` directly, to be inline with verbs' support of `X-Disable-Provenance`